services/backend/src/db/seedData.js

```diff
@@ -8,9 +8,27 @@ const USERS = [
     }
 ];
 
+const PROFILES = [
+    {
+        id: "52e89288-f5f7-43aa-b4cd-0b9e7d6ee458",
+        userId: "3e26fac0-7786-40f3-86bb-f175ff3d721d",
+        name: "test@test.com",
```

Comment:
Not to be pedantic or anything (this is totally pedantic...), but shouldn't the test user's name be... a name?? Like, "Test Test" would make more sense than an email.